src/dotnet/APIView/ClientSPA/angular.json

```diff
@@ -123,5 +123,8 @@
         }
       }
     }
+  },
```

Comment:
Can you remove this

Reply:
Removing either/both of these produces a red squiggle under "cli" and a yellow one under "analytics"

  },
  "cli": {
    "analytics": false
  }
}